src/sentry/new_migrations/migrations.py

```diff
@@ -35,7 +37,7 @@ def validate_operation(op, allow_run_sql):
     if allow_run_sql:
         return
 
-    if isinstance(op, RunSQL):
+    if isinstance(op, RunSQL) and not isinstance(op, SafeRunSQL):
         raise UnsafeOperationException(
             "Using RunSQL is unsafe because our migrations safety framework can't detect problems with the "
```

Comment:
Should this error message also mention using `SafeRunSQL` to ensure that timeouts are applied?

Reply:
Yeah, I'll follow up with that so that we can unblock the current pr